benefits.md

```diff
@@ -99,15 +99,16 @@ Base Two provides 12 paid holidays in which the Base Two office will be closed.
 1. New Year's Day (January 1st)
 2. Martin Luther King Day (3rd Monday in January)
 3. Presidents' Day (3rd Monday in February)
-4. Earth Day (April 22nd or nearest work day)
```

Comment:
❓ Why remove the "or nearest work day" language?

Reply:
🤷‍♂️  b/c none of the others had it (and it's stated below on ln 113).